thing-url-adapter.js

```diff
@@ -464,13 +479,10 @@ class ThingURLDevice extends Device {
 
   performAction(action) {
     action.start();
-
+    const headers = this.adapter.getHeaders(this.actionsUrl);
```

Comment:
This one _does_ need the Content-Type header.

Reply:
added